collectors/statkey.yaml

```diff
@@ -36,3 +36,12 @@ DatastoreStatsCollector:
   - label: "diskspace_capacity_gigabytes"
     statkey: "diskspace|capacity"
 
+VMStatsCollector:
+  - label: "number_of_vcpus_total"
+    statkey: "config|hardware|num_Cpu"
+  - label: "vm_memory_in_kilobytes"
+    statkey: "mem|guest_provisioned"
+  - label: "vm_power_info"
+    statkey: "sys|poweredOn"
```

Comment:
Why are we querying this metric? Is it not already present in the query for properties? 
`property: "summary|runtime|powerState"`

Or are they different?

Reply:
Yes, Almost both are same, one returns a number value..I will remove that.